syntax = "proto3";

package limdongjin.ignasr.protos;

option java_outer_classname = "UserPendingProto";
option java_package = "limdongjin.ignasr.protos";

message UserPending {
  string reqId = 1;
  string userId = 2;
}
